libs/backend/src/cast_vote_records/build_report_metadata.test.ts

```diff
@@ -196,8 +201,6 @@ test('builds well-formed cast vote record report', () => {
       ])
     );
   }
-
-  jest.useRealTimers();
```

Comment:
Moved this up to an `afterEach` because a failed assertion would mean this never gets called.